app/lib/annotation_viz_service.rb

```diff
@@ -47,7 +93,7 @@ def self.get_study_annotation_options(study, user)
   end
 
   # returns a flat array of annotation objects, with name, scope, annotation_type, and values for each
-  def self.available_annotations(study, cluster, current_user, annotation_type=nil)
+  def self.available_annotations(study, cluster=nil, current_user=nil, annotation_type=nil)
```

Comment:
I would convert this to kwargs and refactor where used - having numerous `nil` defaults in a row means you have to make method calls like this:
```AnnotationVizService.get_selected_annotation(@study, nil, nil, nil, 'study')```

I would posit that this is cleaner:

```
def self.available_annotations(study, cluster: nil, current_user: nil, annotation_type: nil)
...
AnnotationVizService.get_selected_annotation(@study, annotation_type: 'study')
```